people/methods/tag.js

```diff
@@ -0,0 +1,115 @@
+var techwords=[ "Access" ,"Account", "Android" ,"Archive" , "Artifical" ,"Application" ,"blog" ,
```

Comment:
make a config.js file and put the words there load these words in the app context and then call here